belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/i18n/I18N.java

```diff
@@ -0,0 +1,151 @@
+package io.github.belgif.rest.problem.i18n;
+
+import java.util.Iterator;
+import java.util.Locale;
+import java.util.ResourceBundle;
+import java.util.ServiceLoader;
+
+import io.github.belgif.rest.problem.api.Problem;
+
+/**
+ * Helper class for I18N (Internationalization).
+ */
+public class I18N {
+
+    public static final String I18N_FLAG = "io.github.belgif.rest.problem.i18n";
+
+    /**
+     * The default locale: English.
+     */
+    public static final Locale DEFAULT_LOCALE = Locale.forLanguageTag("en");
+
+    /**
+     * The Belgif default resource bundle.
+     */
+    public static final String DEFAULT_BUNDLE = "io.github.belgif.rest.problem.Messages";
+
+    private static final LocaleResolver LOCALE_RESOLVER = loadLocaleResolver();
+
+    private static boolean enabled = true;
+
+    static {
+        init();
+    }
+
+    private I18N() {
+    }
+
+    /**
+     * Internal initialization logic. Package accessible for testing purpose.
+     * Checks whether I18N should be enabled, based on system property / environment variable.
+     */
+    static void init() {
+        if (System.getProperty(I18N_FLAG) != null) {
```

Comment:
Isn't this code Java EE specific? For Spring we always let Spring look for the property in all its locations through `@ConfigurationProperties` and set its state here with `I18NConfigurator`. 

The only scenario where this method is relevant is for a Java EE application where it wasn't provided as a servlet context parameter.

Reply:
Moved to java ee I18NConfigurator.